src/Akka.Remote.Hosting/Akka.Remote.Hosting.csproj

```diff
@@ -3,6 +3,8 @@
     <TargetFramework>$(LibraryFramework)</TargetFramework>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <Description>Akka.Remote Microsoft.Extensions.Hosting support.</Description>
+    <LangVersion>9.0</LangVersion>
+    <Nullable>enable</Nullable>
```

Comment:
Can we set this via `Directory.Build.props` and just ensure that this is followed everywhere? Might as well start Akka.Hosting off on the right foot.